libs/measurement-history/src/lib/measurement-history.tsx

```diff
@@ -0,0 +1,140 @@
+import { FilterFilled } from '@ant-design/icons';
+import { DatePicker, Divider } from 'antd';
+import React, { useEffect, useState } from 'react';
+import dayjs, { Dayjs } from 'dayjs';
```

Comment:
Not sure about the Dayjs

Reply:
I'm sure, because the DatePicker.RangePicker (from the ant library) uses dayjs, so I cannot avoid it